src/model/Space.cpp

```diff
@@ -698,7 +698,11 @@ namespace model {
     }
 
     SurfaceVector Space_Impl::surfaces() const {
-      return getObject<ModelObject>().getModelObjectSources<Surface>(Surface::iddObjectType());
+      // get all surfaces, sort so results are repeatable
+      std::vector<Surface> surfaces = getObject<ModelObject>().getModelObjectSources<Surface>(Surface::iddObjectType());
+      std::sort(surfaces.begin(), surfaces.end(), IdfObjectNameLess());
```

Comment:
Is there a significant performance hit by putting the sort here? If so, we can just sort from within the `floorPrint` and `floorArea` methods.

Reply:
There could be a lot of surfaces in a model, but hard to imagine that there would that many for a single space.